runtime/appruntime/api/capture.go

```diff
@@ -19,9 +20,12 @@ type rawRequestBodyCapturer struct {
 	state      captureState
 	underlying io.ReadCloser
 
-	bufMu      sync.Mutex
-	buf        *bytes.Buffer
-	overflowed bool // whether the buffer overflowed
+	bufMu       sync.Mutex
+	buf         *bytes.Buffer
+	overflowed  bool        // whether the buffer overflowed
+	httpMethod  string      // The HTTP method of the request
+	headers     http.Header // The headers of the request
+	queryString url.Values  // The query string of the request captured
```

Comment:
Pretty sure these are all available through other means already, why do they need to be added here?

Reply:
I don't think it is for raw endpoints